nbextensions/bulkImportCell/bulkImportCellStates.js

```diff
@@ -8,13 +8,10 @@
  *      the configuration, this just means it's ready to go.
  * launching - a user clicked "run" and is waiting on a response from the server. This should be
  *      cancelable.
- * queued - the jobs have launched, but all are either in the queue, or otherwise have no state
- *      that's worth looking at
- * running - the jobs are running and have a job state / logs to look at
- * appPartialComplete - at least one job is finished with results to look at
- * appComplete - all of the jobs are complete
- * appCanceled - the cell has canceled its running. Any jobs that have finished can still be looked
- *      at, and logs are still available
+ * inProgress - the jobs have launched and at least one is queued or running; none have completed yet
+ * inProgressResultsAvailable - at least one job is queued or running and at least one job has completed successfully, yielding results
+ * jobsFinished - all jobs are in a terminal state (error/terminated), but none completed successfully
+ * jobsFinishedResultsAvailable - all jobs are in a terminal state, and at least one job completed successfully
```

Comment:
I've edited the FSM states as we discussed earlier, with the addition of a `jobsFinishedResultsAvailable` state so that we can differentiate between where all jobs have finished and none were successful vs when there are some results present.